assets/ui/environmentInfoScreen.ui

```diff
@@ -0,0 +1,58 @@
+{
+    "type": "EnvironmentInfoScreen",
+    "contents": {
+        "type": "relativeLayout",
+        "contents": [
+            {
+                "type": "UIBox",
+                "layoutInfo": {
+                    "width": 300,
+                    "height": 300,
+                    "position-horizontal-center": {},
+                    "position-vertical-center": {}
+                },
+                "content": {
+                	"type" : "relativeLayout",
+					"contents": [
```

Comment:
Whoops, some weird indentation here! Please autoformat this, as well as the Java file, in your IDE.